pkg/cli/clients/clients.go

```diff
@@ -227,8 +227,17 @@ type ApplicationsManagementClient interface {
 	// GetResourceProviderSummary gets the resource provider summary with the specified name in the configured scope.
 	GetResourceProviderSummary(ctx context.Context, planeName string, providerNamespace string) (ucp_v20231001preview.ResourceProviderSummary, error)
 
+	// CreateOrUpdateResourceType creates or updates a resource type in the configured scope.
+	CreateOrUpdateResourceType(ctx context.Context, planeName string, providerNamespace string, resourceTypeName string, resource *ucp_v20231001preview.ResourceTypeResource) (ucp_v20231001preview.ResourceTypeResource, error)
```

Comment:
is the planeName supposed to capture the plane instance too? like /plane/radius/local?

Reply:
Plane name is just the name (`local` in your example). These are Radius-specific APIs we're not letting you specify a plane type (`radius` in your example).